<?xml version="1.0" encoding="UTF-8"?>
<testng-results retried="1" ignored="1" total="7" passed="4" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-09-07T20:46:10 IST" name="Suite" finished-at="2024-09-07T20:47:01 IST" duration-ms="50858">
    <groups>
      <group name="ErrorHandling">
        <method signature="ErrorValidations.LoginErrorValidation()[pri:0, instance:hemanth.tests.ErrorValidations@2dca0d64]" name="LoginErrorValidation" class="hemanth.tests.ErrorValidations"/>
      </group> <!-- ErrorHandling -->
      <group name="Purchase">
        <method signature="StandAloneTestLatest.StandAloneTestLatest(java.util.HashMap)[pri:0, instance:hemanth.tests.StandAloneTestLatest@6ee6f53]" name="StandAloneTestLatest" class="hemanth.tests.StandAloneTestLatest"/>
      </group> <!-- Purchase -->
    </groups>
    <test started-at="2024-09-07T20:46:10 IST" name="Validation Test" finished-at="2024-09-07T20:46:39 IST" duration-ms="29201">
      <class name="hemanth.tests.ErrorValidations">
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:hemanth.tests.ErrorValidations@2dca0d64]" started-at="2024-09-07T20:46:10 IST" name="launchApplication" finished-at="2024-09-07T20:46:13 IST" duration-ms="2525" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method retried="true" signature="LoginErrorValidation()[pri:0, instance:hemanth.tests.ErrorValidations@2dca0d64]" started-at="2024-09-07T20:46:13 IST" name="LoginErrorValidation" finished-at="2024-09-07T20:46:14 IST" duration-ms="1093" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Incorrect email or password.] but found [Incorrect email r password.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Incorrect email or password.] but found [Incorrect email r password.]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at hemanth.tests.ErrorValidations.LoginErrorValidation(ErrorValidations.java:27)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginErrorValidation -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:hemanth.tests.ErrorValidations@2dca0d64]" started-at="2024-09-07T20:46:14 IST" name="tearDown" finished-at="2024-09-07T20:46:14 IST" duration-ms="249" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:hemanth.tests.ErrorValidations@2dca0d64]" started-at="2024-09-07T20:46:14 IST" name="launchApplication" finished-at="2024-09-07T20:46:16 IST" duration-ms="1898" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="LoginErrorValidation()[pri:0, instance:hemanth.tests.ErrorValidations@2dca0d64]" started-at="2024-09-07T20:46:16 IST" name="LoginErrorValidation" finished-at="2024-09-07T20:46:16 IST" duration-ms="431" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Incorrect email or password.] but found [Incorrect email r password.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Incorrect email or password.] but found [Incorrect email r password.]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at hemanth.tests.ErrorValidations.LoginErrorValidation(ErrorValidations.java:27)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:269)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:76)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginErrorValidation -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:hemanth.tests.ErrorValidations@2dca0d64]" started-at="2024-09-07T20:46:17 IST" name="tearDown" finished-at="2024-09-07T20:46:17 IST" duration-ms="200" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:hemanth.tests.ErrorValidations@2dca0d64]" started-at="2024-09-07T20:46:17 IST" name="launchApplication" finished-at="2024-09-07T20:46:19 IST" duration-ms="1823" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="ProductErrorValidation()[pri:0, instance:hemanth.tests.ErrorValidations@2dca0d64]" started-at="2024-09-07T20:46:19 IST" name="ProductErrorValidation" finished-at="2024-09-07T20:46:39 IST" duration-ms="20441" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ProductErrorValidation -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:hemanth.tests.ErrorValidations@2dca0d64]" started-at="2024-09-07T20:46:39 IST" name="tearDown" finished-at="2024-09-07T20:46:39 IST" duration-ms="223" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- hemanth.tests.ErrorValidations -->
    </test> <!-- Validation Test -->
    <test started-at="2024-09-07T20:46:10 IST" name="Submit Order" finished-at="2024-09-07T20:47:01 IST" duration-ms="50858">
      <class name="hemanth.tests.StandAloneTestLatest">
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:hemanth.tests.StandAloneTestLatest@6ee6f53]" started-at="2024-09-07T20:46:10 IST" name="launchApplication" finished-at="2024-09-07T20:46:13 IST" duration-ms="2756" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="StandAloneTestLatest(java.util.HashMap)[pri:0, instance:hemanth.tests.StandAloneTestLatest@6ee6f53]" started-at="2024-09-07T20:46:13 IST" name="StandAloneTestLatest" data-provider="getData" finished-at="2024-09-07T20:46:35 IST" duration-ms="21560" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{password=Keerthana077@, email=Keerthana077@gmail.com, productName=IPHONE 13 PRO}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- StandAloneTestLatest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:hemanth.tests.StandAloneTestLatest@6ee6f53]" started-at="2024-09-07T20:46:35 IST" name="tearDown" finished-at="2024-09-07T20:46:35 IST" duration-ms="224" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:hemanth.tests.StandAloneTestLatest@6ee6f53]" started-at="2024-09-07T20:46:35 IST" name="launchApplication" finished-at="2024-09-07T20:46:36 IST" duration-ms="1493" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="StandAloneTestLatest(java.util.HashMap)[pri:0, instance:hemanth.tests.StandAloneTestLatest@6ee6f53]" started-at="2024-09-07T20:46:36 IST" name="StandAloneTestLatest" data-provider="getData" finished-at="2024-09-07T20:46:58 IST" duration-ms="21724" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{password=Keerthana077@, email=Keerthana077@gmail.com, productName=ADIDAS ORIGINAL}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- StandAloneTestLatest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:hemanth.tests.StandAloneTestLatest@6ee6f53]" started-at="2024-09-07T20:46:58 IST" name="tearDown" finished-at="2024-09-07T20:46:59 IST" duration-ms="297" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:hemanth.tests.StandAloneTestLatest@6ee6f53]" started-at="2024-09-07T20:46:59 IST" name="launchApplication" finished-at="2024-09-07T20:47:00 IST" duration-ms="1598" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="orderHistoryTest()[pri:0, instance:hemanth.tests.StandAloneTestLatest@6ee6f53]" started-at="2024-09-07T20:47:00 IST" name="orderHistoryTest" finished-at="2024-09-07T20:47:01 IST" depends-on-methods="hemanth.tests.StandAloneTestLatest.StandAloneTestLatest" duration-ms="726" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- orderHistoryTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:hemanth.tests.StandAloneTestLatest@6ee6f53]" started-at="2024-09-07T20:47:01 IST" name="tearDown" finished-at="2024-09-07T20:47:01 IST" duration-ms="202" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- hemanth.tests.StandAloneTestLatest -->
    </test> <!-- Submit Order -->
  </suite> <!-- Suite -->
</testng-results>
